src/pyodide/internal/_workers.py

```diff
@@ -938,18 +964,16 @@ class _WorkflowStepWrapper:
     def __init__(self, js_step):
         self._js_step = js_step
 
-    def do(self, name, callback=None, config=None):
-        if callback:
-
-            async def wrapper():
-                return _do_call(self, name, config, callback)
-
-            return wrapper()
-
-        # Return a decorator that will execute the function when called
+    def do(self, name, depends=None, concurrent=False, config=None):
         def decorator(func):
+            @functools.wraps(func)
```

Comment:
Again I think this doesn't belong here because `@functools.wraps()` is for when we're not changing the arguments but the wrapper always takes zero arguments.